### ISSUE TITLE
[DRC-535] [Bug] CLI command `Summary` doesn't have dbt params

### ISSUE THREAD

**Issue by jaklau** [DataRecce/recce#372]
### Current Behavior

When using `recce summary recce_state.json` I get an error:
`Cannot initiate the dbt project. Reason: No dbt_project.yml found at expected path {path_name}/data/dbt_project.yml
`

But there is no dbt flags in `recce summary`.

### Expected Behavior

`recce summary` have dbt params as in `recce run`

### Steps To Reproduce

1. Run `recce summary recce_state.json` outside of the dbt project.

### Relevant log output

```shell
Cannot initiate the dbt project. Reason: No dbt_project.yml found at expected path {path_name}
Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
```

### Environment

- recce: 23.0.21
- OS: MacOS 14.5
- Python: 3.10
- Data Warehouse: BigQuery
- dbt: 1.7.10

### Additional Context

Maybe it would only need to add `@add_options(dbt_related_options)` decorator to https://github.com/DataRecce/recce/blob/main/recce/cli.py#L310

<sub>[DRC-535](https://linear.app/piperider/issue/DRC-535/[bug]-cli-command-summary-doesnt-have-dbt-params)</sub>

**Comment by popcornylu** [comment]
Hi, @jaklau. Thanks for your report.

This is indeed a problem, and we will solve it. Ideally,  it should be supported to run `recce summary recce_state.json` outside of the dbt project.

**Comment by popcornylu** [comment]
Fixed in [v0.24.0](https://github.com/DataRecce/recce/releases/tag/v0.24.0)
